### PR TITLE
Change error to the scheduler error

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

### CODE REVIEW

File: task-ballerina/listener.bal [hunk 1]
```diff
@@ -107,34 +107,34 @@ public class Listener {
     # Pauses the `task:Listener` and the attached services.
     #
     # + return - A `task:ListenerError` if an error occurred while pausing or else ()
-    public isolated function pause() returns ListenerError? {
+    public isolated function pause() returns error? {
```
Comment (daneshk) [inline task-ballerina/listener.bal:110]: Why do we need to change `ListenerError` to `error`? specific reason.

Comment (kalaiyarasiganeshalingam) [reply task-ballerina/listener.bal:110]: I have got the following error from the isolated function when running the negative test cases. It means if we getting any error from the java level, can't be cast to ballerina specific error. So I have changed to a common error.
` {ballerina}TypeCastError {"message":"incompatible types: 'error' cannot be cast to 'task:ListenerError|()'"}`

Comment (ldclakmal) [reply task-ballerina/listener.bal:110]: According to the message, it seems like, the type `error` is not equivalent to the type `ListenerError`. Shall we verify the reason for that?

Comment (kalaiyarasiganeshalingam) [reply task-ballerina/listener.bal]: Yes, got this error by a different native package version.

**Review by daneshk — APPROVED** [review]

**Pull request merged**